jbpm-test-coverage/src/test/java/org/jbpm/test/functional/async/AsyncAdHocSubprocessTest.java

```diff
@@ -91,6 +126,81 @@ public void testAdHocSubAsyncProcessAuto() throws Exception {
         assertProcessInstanceAborted(id);
     }
 
+    @Test(timeout = 30000)
```

Comment:
Isn't it too much? Actually child process 3, just directly throw a runtimeException so it should be pretty straight forward.

Reply:
it is the standard around